fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingBase.java

```diff
@@ -613,6 +617,13 @@ public static PartlyBuiltException getAPartlyBuildExceptionIfApplicable(@Nullabl
         return null;
     }
 
+    protected List<Object> runAsyncDetails(String transactionName, Object ...keysAndValues) {
+        return new ImmutableList.Builder<>()
+                .add(LogMessageKeys.TRANSACTION_NAME, transactionName)
+                .add(common.indexLogMessageKeyValues())
```

Comment:
Instead of the wrapping function `runAsyncDetails`, wouldn't it make sense to use `common.indexLogMessageKeyValues(String transactionName)`? 
(If `Object ...keysAndValues` is never used, maybe we can wait with the implementation until it's needed)